po/appstream/en_CA.po

```diff
@@ -9,7 +9,7 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: PACKAGE version\n"
```

Comment:
This shouldn't have been changed ;)

Reply:
A little fckup provided by PyCharm (: Gimme 1min